src/aiortc/contrib/media.py

```diff
@@ -540,20 +641,32 @@ class MediaRecorder:
     :param options: Additional options to pass to FFmpeg.
     """
 
-    def __init__(self, file, enable_prediction=False, reference_update_freq=30, 
-                output_fps=30, format=None, save_dir=None, options={}):
+    def __init__(self, file, enable_prediction=False, prediction_type="keypoints",
+                reference_update_freq=30, output_fps=30, format=None, save_dir=None, options={}):
         self.__container = av.open(file=file, format=format, mode="w", options=options)
         self.__received_keypoints_frame_num = 0
         self.__keypoints_file_name = str(file).split('.')[0] + "_recorded_keypoints.txt"
         self.__tracks = {}
         self.__frame_height = None
         self.__frame_width = None
         self.__keypoints_queue = asyncio.Queue()
-        self.__video_queue = asyncio.Queue()
+        self.__source_queue = asyncio.Queue()
+        self.__real_video_queue = asyncio.Queue()
+        self.__synthatic_video_queue = asyncio.Queue()
+        self.__lr_video_queue = asyncio.Queue()
+        self.__display_queue = asyncio.Queue()
         self.__save_dir = save_dir
         self.__enable_prediction = enable_prediction
+        self.__prediction_type = prediction_type
         self.__reference_update_freq = reference_update_freq
         self.__output_fps = output_fps
+        self.__set_video_size = True
+        self.__display_option = "synthatic" #"real" #"synthatic"
+        '''
+        __display_option could be:
+            1. "real": vpx only
```

Comment:
is this a one-time setting or modified through the course of the video

Reply:
if it is modified, I wouldn't use "vpx only" in the comments

Reply:
This could change based on the feedback given to the application layer, I changed the description